eng/versioning/external_dependencies.txt

```diff
@@ -22,7 +22,6 @@ com.microsoft.azure:spotbugs-maven-plugin;1.2.1
 com.microsoft.rest:client-runtime;1.6.15
 com.microsoft.rest.v2:client-runtime;2.1.1
 com.squareup.okhttp3:okhttp;4.2.2
-commons-codec:commons-codec;1.13
```

Comment:
Your PR is failing because you removed an external_dependency that is used by other libraries. In the future, if you're curious as to whether a dependency is being used you can run one of two quick commands to check:
- From a powershell 6 window, in the root of the enlistment run .\eng\pom_file_version_scanner.ps1 (thought this is only going to tell you after you remove the entry)
- For something quick and dirty on Windows you can just run findstr /spin /c:"commons-codec:commons-codec" pom*.xml

Reply:
yep, I already reverted this change